nav2_bringup/bringup/params/nav2_multirobot_params_1.yaml

```diff
@@ -65,6 +65,9 @@ bt_navigator:
     - nav2_recovery_node_bt_node
     - nav2_pipeline_sequence_bt_node
     - nav2_transform_available_condition_bt_node
+    - nav2_is_new_goal_condition_bt_node
```

Comment:
Make sure to update the default list in the nav2 behavior tree server as well in case no param file is given.